models/src/main/scala/coop/rchain/models/either/EitherHelper.scala

```diff
@@ -0,0 +1,48 @@
+package coop.rchain.models.either
+
+import coop.rchain.either.{EitherError, EitherSuccess}
+import coop.rchain.either.Either.Content
+import coop.rchain.models.StacksafeMessage
+import coop.rchain.shared.ThrowableOps._
+
+import scalapb.GeneratedMessageCompanion
+
+object EitherHelper {
+  def toEither[A <: StacksafeMessage[A]](
+      msg: coop.rchain.either.Either
+  )(implicit cmp: GeneratedMessageCompanion[A]): Either[Seq[String], A] = {
+    require(msg.content.isDefined, "Either message has no content")
```

Comment:
Does this `require` halt the node if it fails? Does that mean I can bring down a node by sending it a bad GRPC message?

Reply:
@dckc No, this is used on the client side. The client will fail if the service (node) replies with a bad grpc message.